crypto/evp/evp_locl.h

```diff
@@ -39,6 +39,7 @@ struct evp_cipher_ctx_st {
     int final_used;
     int block_mask;
     unsigned char final[EVP_MAX_BLOCK_LENGTH]; /* possible final block */
+    RAND_DRBG *drbg;
```

Comment:
Does it make sense to not have "just" the DRBG, but the whole SSL object?  I can see the arguments for either way.